src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/LaunchProfilesDebugLaunchProvider.cs

```diff
@@ -113,6 +114,34 @@ private async Task<IReadOnlyList<IDebugLaunchSettings>> QueryDebugTargetsInterna
             return null;
         }
 
+        private class LaunchCompleteCallback : IVsDebuggerLaunchCompletionCallback
+        {
+            private readonly DebugLaunchOptions _launchOptions;
+            private readonly IDebugProfileLaunchTargetsProvider? _targetProfile;
+            private readonly ILaunchProfile? _activeProfile;
+
+            public LaunchCompleteCallback(DebugLaunchOptions launchOptions, IDebugProfileLaunchTargetsProvider? targetProfile, ILaunchProfile? activeProfile)
+            {
```

Comment:
`activeProfile` can never be null if I'm reading the code correctly, mark it non-nullable and remove the assertion below.